web/elm/Dream/New/View.elm

```diff
@@ -0,0 +1,24 @@
+module Dream.New.View exposing (view)
+
+import Html exposing (..)
+import Html.Attributes exposing (class, placeholder, required, value)
+import Html.Events exposing (onClick, onInput)
+
+import Dream.Types exposing (Msg(Create, ChangeContent))
+import Dream.New.Types exposing (Dream)
+
+view : Dream -> Html Msg
+view dream =
+  div [ class "input-group" ] [
+    input [
+      placeholder "What are you dreaming about?",
+      required True,
+      class "form-control",
+      value dream.content,
+      onInput ChangeContent
+    ] [],
+    span [ class "input-group-btn" ]
+      [
```

Comment:
this is hella nitpicky, but if you move this opening bracket up to the end of your `span` line (like you do with `div` on line 12), you can avoid the double un-indent at the end of this